### DISCUSSION TITLE
Add an aliases: `clock` → `on`, `fn` → `map` in `sample`?

### DISCUSSION THREAD

**Discussion by demark** [zerobias/effector#737] [Ideas]
Hi folks, before opening an issue, I wanted to discuss adding an aforementioned aliases in `sample` for consistency and easier adoption across various teams experienced with other community-popular state/flow managers. I've just read #721 brought up by @igorkamyshev and kinda agree that Effector uses some uncommon api terms that makes it more difficult for onboarding. Adding these aliases should be straightforward and non-breaking.

Here is the rationale:

1. **Alias for `clock` → `on`**. Firstly, this makes it easier for adoption for newcomers due to common use of `on` for any sort of triggers. Secondly, it would look consistent with `$store.on(trigger, ...)` api. I have seen (and used) dozen of state/flow managers and have never seen usage of `clock` term for triggers.
2. **Alias for `fn` → `map`**. Although it is described as combinator function, it could be _conceptually_ described as a mapper function. And again, `map` is common, widely used and familiar way to transform data. Finally, that _looks_ consistent with `$store.map(...)` (but not the same though).

Any thoughts?

**Comment by AlexandrHoroshih** [comment]
I think, i agree with @sergeysova arguments in the #721 🤔 

Just changing current terms to more common ones will require a lot of work on correcting docs, providing explanations about this transition - looks like it will create more confusion since there will be two words for the same thing 🤔 

Also many reactive libraries do have their own "language" - e.g. `react`  with its `hooks`, `props`,  `renders`, special understanding of `effects`, etc.
In case of react it is handled by awesome official documentation (which is basically developed by special team) and lots of entry-level articles all over the internet.

**Comment by AlexandrHoroshih** [comment]
I think, that issue with effector "language" also can be solved by documentation 🤔 

e.g. if we add "Concepts" section in the docs with explanations on what is a `clock` or `source` - it will help new users with understanding of api terms and all of the existing content will remain actual, without confusing notes about aliases
